### PR TITLE
Test Suite for Week 1 code

### LINKED ISSUE #19: Create test suite for first week code
As a user, I want hammer-cli to be well tested. With limited development time in our first week, we need to catch up on test cases. Also, a code coverage tool should be set up to help us maintain good tests. 

- [ ] Verify a code coverage tool is set up
- [ ] Verify tests are well written for code written in the first week

### PR DESCRIPTION
**Pull request by jackcmeyer**
closes #19 
* added tests for initProject()

### CODE REVIEW

**Comment by jackcmeyer** [comment]
@hammer-io/developers can you all take a look at this to make sure this is good test coverage and make sure I didn't miss anything?

File: test/cli-test.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  description: 'Jack\'s Test Project',
+  version: '0.0.1',
+  author: 'Jack Meyer',
+  license: 'MIT',
+  ci: 'TravisCI',
+  container: 'Docker'
+};
+
+describe('Init Project', () => {
+  describe('#initProject()', () => {
```
Comment (nischayv) [inline test/cli-test.js]: You could avoid having excessive describe blocks

File: test/cli-test.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+});
+
+describe('Init Project', () => {
+  describe('#initProject()', () => {
+    it('should create a src directory underneath the project directory', () => {
+      cli.initProject(configs);
+      assert.equal(true, fs.existsSync(configs.projectName + '/src'));
+
+      // clean up directory that was created
+      fsExtra.removeSync(configs.projectName);
```
Comment (nischayv) [inline test/cli-test.js]: This could be put in an after each block instead of calling it in every test

Comment (KrashLeviathan) [reply test/cli-test.js]: Along that same line of thought, is there a way to put the `cli.initProject(configs);` in something to run before each block?

Comment (jackcmeyer) [reply test/cli-test.js]: I've made a change to add a before and after block.

File: test/cli-test.js (hunk 1 above)
Comment (jackcmeyer) [reply test/cli-test.js]: I've changed this.

**Pull request merged**